scripts/util.ts

```diff
@@ -0,0 +1,95 @@
+import BN from "bn.js";
+import type { BatchExchangeInstance } from "../types/truffle-typings";
+import { factory } from "../src/logging";
+const log = factory.getLogger("scripts.util");
+
+export interface ExchangeToken {
+  id: number;
+  address: string;
+}
+
+export async function getBatchExchange(
```

Comment:
I think these should probably be removed from their JS counterpart. Note that if they are still being used by other JS scripts, they can `const { getBatchExchange } = require("../build/common/scripts/util")`.